app/src/main/java/com/xa/xpensauditor/SetLimitActivity.java

```diff
@@ -0,0 +1,286 @@
+package com.xa.xpensauditor;
+
+import static com.xa.xpensauditor.AddTransactionActivity.printLog;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+
+import java.lang.ref.Reference;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.lang.ref.Reference'